source/extensions/filters/http/common/aws/utility.cc

```diff
@@ -123,11 +128,12 @@ absl::optional<std::string> Utility::metadataFetcher(const std::string& host,
   }
 
   for (size_t retry = 0; retry < MAX_RETRIES; retry++) {
+    ENVOY_LOG_TO_LOGGER(logger, debug, "Fetching AWS metadata from {}", url);
     const CURLcode res = curl_easy_perform(curl);
     if (res == CURLE_OK) {
       break;
     }
-    ENVOY_LOG_MISC(debug, "Could not fetch AWS metadata: {}", curl_easy_strerror(res));
+    ENVOY_LOG_TO_LOGGER(logger, warn, "Could not fetch AWS metadata: {}", curl_easy_strerror(res));
```

Comment:
I think you probably want a stat here of some type? Are you worried also about log spam here? How often would this log during failure?